### PR TITLE
fix: change to use bundleid filter app

### PR DESCRIPTION
**Pull request by Fatpandac**
## Description

Change the filtering of the terminal app to use bundle ID, in order to support systems in other languages.

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [X] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [X] I checked that files in the `assets` folder are used by the extension itself
- [X] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by louishuyng** [comment]
Cool Update @Fatpandac 👍

@pernielsentikaer please help to check and approve this PR 🙏

**Review by louishuyng — APPROVED** [review]

File: extensions/tmux-sessioner/package-lock.json [hunk 1]
```diff
@@ -1,2092 +1,7 @@
...
-      "dev": true
-    },
-    "node_modules/yallist": {
-      "version": "4.0.0",
-      "resolved": "https://registry.npmjs.org/yallist/-/yallist-4.0.0.tgz",
-      "integrity": "sha512-3wdGidZyq5PB084XLES5TpOSRA3wjXAlIWMhum2kRcv/41Sn2emQ0dycQW4uZXLejwKvg6EsvbdlVL+FYEct7A==",
-      "dev": true
-    }
-  },
+  "lockfileVersion": 1,
```
[the remark this answers is not in the document]
Comment (Fatpandac) [reply extensions/tmux-sessioner/package-lock.json]: You are right, it has been fixed in the latest commit.

File: extensions/tmux-sessioner/src/utils/terminalUtils.ts [hunk 2]
```diff
@@ -25,5 +25,5 @@ export async function checkTerminalSetup(callback: (isTerminalSetup: boolean) =>
 
 export async function openTerminal() {
   const localTerminalAppName = await LocalStorage.getItem<string>("terminalAppName");
-  execSync(`open -a ${localTerminalAppName}`);
+  execSync(`open -b ${localTerminalAppName}`);
```
Comment (louishuyng) [inline extensions/tmux-sessioner/src/utils/terminalUtils.ts:28]: @Fatpandac  i think we should change the name to align with this behavior.

```suggestion
  execSync(`open -b ${localTerminalAppBundleId}`);
```


Also in the form we may need to rename the dropdown ID also

```typescript
<Form.Dropdown id="terminalAppBundleId" isLoading={loading}>
```

Comment (Fatpandac) [reply extensions/tmux-sessioner/src/utils/terminalUtils.ts:28]: Yes, but I will fix it tomorrow night. I am going to bed now.

Comment (louishuyng) [reply extensions/tmux-sessioner/src/utils/terminalUtils.ts:28]: Just take your time 👍 no rush in here and rest well

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**